lib/__tests__/does-not-exist.ts

```diff
@@ -76,10 +76,14 @@ describe('assert.dom(...).doesNotExist()', () => {
       'Unexpected Parameter: [object HTMLBodyElement]'
     );
 
+    //@ts-ignore -- These assertions are for JavaScript users who don't have type checking
```

Comment:
Just a thought about this, but I wonder if it's better to extract these tests out to a js file rather than have to ignore them, when they're clearly aimed at testing the looser allowances of js. 

This would have to be repeated for all below, thought we _could_ put these in a separate subdirectory in tests.

What do you think?

Reply:
seems harder to maintain to me, so I'll go ahead an merge as-is :)